prowler/providers/aws/services/kinesis/kinesis_service.py

```diff
@@ -0,0 +1,75 @@
+from enum import Enum
+
+from pydantic import BaseModel
+
+from prowler.lib.logger import logger
+from prowler.lib.scan_filters.scan_filters import is_resource_filtered
+from prowler.providers.aws.lib.service.service import AWSService
+
+
+class Kinesis(AWSService):
+    def __init__(self, provider):
+        # Call AWSService's __init__
+        super().__init__("kinesis", provider)
+        self.streams = {}
+        self.__threading_call__(self._list_streams)
+        self._describe_stream()
+
+    def _list_streams(self, regional_client):
+        logger.info("Kinesis - Listing Kinesis Streams...")
+        try:
+            list_streams_paginator = regional_client.get_paginator("list_streams")
+            for page in list_streams_paginator.paginate():
+                for stream in page["StreamSummaries"]:
+                    if not self.audit_resources or (
+                        is_resource_filtered(stream, self.audit_resources)
+                    ):
+                        self.streams[stream["StreamARN"]] = Stream(
+                            arn=stream["StreamARN"],
+                            name=stream["StreamName"],
+                            region=regional_client.region,
+                            status=StreamStatus(stream.get("StreamStatus", "DELETING")),
+                        )
+        except Exception as error:
+            logger.error(
+                f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+            )
+
+    def _describe_stream(self):
```

Comment:
Why this function? if you already have the status of the stream?

Reply:
I implemented it to retrieve the tags, and I plan to expand its functionality once this PR is merged. I wasn't aware of the redundancy in the status field, but that has been corrected now.